keyboards/oddball/adns.c

```diff
@@ -0,0 +1,270 @@
+#include <avr/io.h>
+#include <avr/interrupt.h>
+#include "quantum.h"
+#include "pointing_device.h"
+#include "adns9800_srom_A4.h"
+#include "../../lib/lufa/LUFA/Drivers/Peripheral/SPI.h"
```

Comment:
Just a heads up, I'm working on an SPI driver in #8299. This should let you not have to reference the LUFA code, in case you wanted to reuse this on Arm :)

Reply:
Cheers, good to know!